docs/Network.md

```diff
@@ -50,6 +77,22 @@ fetch('https://mywebsite.com/endpoint.php')
   });
 ```
 
+2.  Called within an asynchronous function using ES7 `async`/`await` syntax:
```

Comment:
Yes, it's purely implemented in JS. (The iOS bridge has support for defining bridged async functions, which we need to add to Android. But you can use pure JS async functions on both platforms.)